pkg/tests/end_to_end_tests/integration_test.go

```diff
@@ -0,0 +1,390 @@
+// Copyright (c) 2019 The Jaeger Authors.
```

Comment:
Should we add a prefix to the file name to specify that's a jaeger integration test?

Reply:
Done.